modules/t/citest_division/compara_citest.xml

```diff
@@ -35,14 +46,21 @@
   </pairwise_alignments>
 
   <multiple_alignments>
-    <!-- Mammals, excl. the mouse strains -->
+    <!-- Mammals -->
     <multiple_alignment method="EPO" gerp="1">
       <species_set name="mammals" display_name="eutherian mammals">
         <taxonomic_group taxon_name="Eutheria"/>
       </species_set>
     </multiple_alignment>
 
-    <!-- Amniotes, excl. mouse strains -->
+    <!-- Mammals + Chicken -->
+    <multiple_alignment method="EPO_LOW_COVERAGE" gerp="1">
+      <species_set name="amniotes" display_name="mammals with feathers" in_collection="mammals_with_feathers">
```

Comment:
You can remove the `in_collection` attribute since the "Amniota" filter will still return the same set when applied to the entire "citest" division

Reply:
Unless you really want to call this mlss "mammals with feathers", you can remove the `display_name` attribute too. create_all_mlss will use "amniotes" instead